### PR TITLE
Update a link to the QuestDB integration

### PR DESCRIPTION
**Pull request by ieshreya**
## Description

Update a link to the QuestDB integration

**Fixes** #4823

## Type of change

(Please delete options that are not relevant)

- [x] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

File: README.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ MindsDB works with most of the SQL and NoSQL databases and data Streams for real
...
-| <a href="https://docs.mindsdb.com/"><img src="https://img.shields.io/badge/QuestDB-d14671?style=for-the-badge&logo=questdb&logoColor=white" alt="Connect QuestDB"></a> | <a href="https://docs.mindsdb.com/"><img src="https://img.shields.io/badge/Apache%20Druid-29F1FB?logo=apachedruid&logoColor=000&style=for-the-badge" alt="Apache Druid Badge"></a> | <a href="https://docs.mindsdb.com/"><img src="https://img.shields.io/badge/starrocks-000000?style=for-the-badge&logo=starrocks&logoColor=white" alt="StarRocks Badge"></a> |
+| <a href="[https://docs.mindsdb.com/](https://docs.mindsdb.com/data-integrations/all-data-integrations#questdb)"><img src="https://img.shields.io/badge/QuestDB-d14671?style=for-the-badge&logo=questdb&logoColor=white" alt="Connect QuestDB"></a> | <a href="https://docs.mindsdb.com/"><img src="https://img.shields.io/badge/Apache%20Druid-29F1FB?logo=apachedruid&logoColor=000&style=for-the-badge" alt="Apache Druid Badge"></a> | <a href="https://docs.mindsdb.com/"><img src="https://img.shields.io/badge/starrocks-000000?style=for-the-badge&logo=starrocks&logoColor=white" alt="StarRocks Badge"></a> |
```
Comment (martyna-mindsdb) [inline README.md:148]: Please correct it as shown below:
```
<a href="https://docs.mindsdb.com/data-integrations/all-data-integrations#questdb"><img src=...
```

**Comment by martyna-mindsdb** [comment]
@ieshreya
Please implement the changes as requested in the review comment above.

Also, before we can merge your PR, please sign our CLA by leaving the following comment here: I have read the CLA Document and I hereby sign the CLA

Thanks.

**Comment by ieshreya** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**